lib/resources/nginx_conf.rb

```diff
@@ -92,4 +101,92 @@ def resolve_references(data, rel_path)
       Hash[data.map { |k, v| [k, resolve_references(v, rel_path)] }]
     end
   end
+
+  class NginxConfHttp
+    attr_reader :entries
+    def initialize(params, parent)
+      @parent = parent
+      @entries = params.map { |x| NginxConfHttpEntry.new(x, parent) }
+    end
+
+    def servers
+      @entries.map(&:servers).flatten
+    end
+
+    def locations
+      servers.map(&:locations).flatten
+    end
+
+    def to_s
+      @parent.to_s + ', http entries'
+    end
+    alias inspect to_s
```

Comment:
we should decide if we want the inspec shell to have the pretty-printed resources or their `inspect` equivalent. for example: `nginx_conf` right now in the inspec shell will pretty-print to `=> nginx_conf /etc/nginx/nginx.conf`. without this change it would print the entire page full of param debug. I aimed for consistency and for less printing, especially on `http`, but i also get the argument for the counterpoint (not to forget fewer changes).